src/providers/ecency/ecency.ts

```diff
@@ -31,9 +31,9 @@ import {
  * ************************************
  */
 
-export const getCurrencyRate = (currency) =>
+export const getFiatHbdRate = (fiatCode:string) =>
   ecencyApi
-    .get(`/private-api/market-data/${currency}/hbd?fixed=1`)
+    .get(`/private-api/market-data/${fiatCode}/hbd?fixed=1`)
```

Comment:
@noumantahir we don't use fixed=1 on website, maybe that's why there is slight difference in prices.

Reply:
seems it has not effect at all... removed it still

![Screenshot 2023-07-21 at 5 42 16 PM](https://github.com/ecency/ecency-mobile/assets/6298342/bd25bde7-04e2-467d-a91a-cde1d3f7fcd7)

Reply:
@noumantahir oh, few days ago I have pushed some fixes to vision-api looks like forgot to handle query params. Let me fix that.

Reply:
@noumantahir fixed, try https://ecency.com/private-api/market-data/usd/hbd?fixed="1"